pom.xml

```diff
@@ -684,6 +695,15 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
                 <version>3.3.1</version>
+<!--               <executions>-->
```

Comment:
There was a problem at build. Maven plugin failed, because it detected that this execution is duplicate, so I commented it out, but this must be fixed, or the sources will not be attached.